build/ubuntu/Dockerfile.deps

```diff
@@ -3,17 +3,20 @@ FROM ubuntu:20.04 as cryptobox-builder
 # compile cryptobox-c
 RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get update && \
-    apt-get install -y cargo file libsodium-dev git pkg-config make && \
+    apt-get install -y cargo file libsodium-dev git pkg-config make curl && \
     cd /tmp && \
     git clone https://github.com/wireapp/cryptobox-c.git && \
     cd cryptobox-c && \
     export SODIUM_USE_PKG_CONFIG=1 && \
     cargo build --release
 
 # compile mls-test-cli tool
+# note: to make use of rustup's toolchain you need to prefix commands by . "$HOME/.cargo/env"
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
not ideal that we rely on a source like rustup.rs. Can we find a ubuntu-official way to get a more up-to-date version